Demo/src/main/java/com/paypal/android/ui/card/CardFragment.kt

```diff
@@ -223,13 +222,6 @@ class CardFragment : Fragment() {
             ),
             payee = Payee(emailAddress = "anpelaez@paypal.com")
         )
-
-        if (shouldRequestThreeDSecure) {
-            createOrderRequest.applicationContext = ApplicationContext(
-                returnURL = APP_RETURN_URL,
-                cancelURL = APP_CANCEL_URL
-            )
-        }
```

Comment:
Originally we were using LSATs and `confirm-payment-source` for this call, but then we had to reach out to that team because there was a bug where we couldn't update the return url after order creation. Once they fixed it we verified with this PR to allow the Client SDK to set the return and cancel URLs.

Temporarily we set them on the sample merchant server as a workaround and we planned on migrating to this method once the fix was in.